httplog.go

```diff
@@ -352,12 +352,12 @@ func LogEntrySetField(ctx context.Context, key string, value slog.Value) {
 
 func LogEntrySetFields(ctx context.Context, fields map[string]interface{}) {
 	if entry, ok := ctx.Value(middleware.LogEntryCtxKey).(*RequestLoggerEntry); ok {
-		attrs := make([]slog.Attr, len(fields))
+		attrs := make([]any, len(fields))
```

Comment:
Is this change really necessary?

Reply:
`*entry.Logger.With(attrs...)` has the following signature:

```
func (l *Logger) With(args ...any)
```

![image](https://github.com/go-chi/httplog/assets/77052806/f7a4a8d7-dc1e-45af-8203-efd6348a9a33)